File: programaçãoMobile/projetos/FrasesdoDia/app/src/main/java/com/example/frasesdodia/MainActivity.java
package com.example.frasesdodia;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    String[] frases = {
            "Frase 1",
            "Frase 2",
            "Frase 3",
    }

    public void gerarFrase(View view){


    }

    public void exibirTodas(View view){

        for ( String frase : frases){
            System.out.println(frase);
        }

    }
}

File: programaçãoMobile/projetos/MeuPrimeiroProjeto/app/src/main/java/com/example/meuprimeiroprojeto/MainActivity.kt
package com.example.meuprimeiroprojeto

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import java.util.Random

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun sortear(view: View){
        val textoResultado = findViewById<TextView>(R.id.text_result)
        val num = Random().nextInt(11)
        textoResultado.setText("Número gerado $num")
    }




}